modules/account-lib/test/unit/coin/algo/utils.ts

```diff
@@ -21,4 +22,25 @@ describe('utils', () => {
     const objEncodedByAlgo = algosdk.encodeObj(note);
     should.deepEqual(objEncoded, objEncodedByAlgo);
   });
+
+  it('should build fee data on Pay Tx and Non Participation Tx', () => {
```

Comment:
This is only available for PayTx and NonParticipationTx? What about KeyRegTx? Do we need extra tests?

Reply:
Ok about that. To make this tests over here I've run MS tests and copy the exactly returned Tx here. So nowdays we only have this two builders on MS which use buildFeeInfo. Keyreg will use it when my other PR is done, so now I cant figure out the spefic output to compare on mocha test.

Reply:
Let me add this